stader/node/manage-fee-recipient.go

```diff
@@ -119,6 +121,33 @@ func (m *manageFeeRecipient) run() error {
 		return err
 	}
 
+	currentBlock, err := eth1.GetCurrentBlockNumber(m.c)
+	if err != nil {
+		return fmt.Errorf("error GetCurrentBlockNumber: %w", err)
+	}
+
+	pnr, err := services.GetPermissionlessNodeRegistry(m.c)
+	if err != nil {
+		return fmt.Errorf("error GetPermissionlessNodeRegistry: %w", err)
+	}
+
+	operatorID, err := node.GetOperatorId(pnr, nodeAccount.Address, nil)
+	if err != nil {
+		return fmt.Errorf("error GetOperatorId: %w", err)
+	}
+
+	lastChangeBlock, err := node.GetSocializingPoolStateChangeBlock(pnr, operatorID, nil)
+	if err != nil {
+		return fmt.Errorf("error GetSocializingPoolStateChangeBlock: %w", err)
+	}
+
+	nextUpdatableBlock := lastChangeBlock.Add(lastChangeBlock, blockPerThreeEpoch).Uint64()
```

Comment:
where is this value coming from? blockPerThreeEpoch

Reply:
Hi Rahul, this value should be: `big.NewInt(96)`. Last commit I missed this.